recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py3k]
+  skip: True  # [osx and py3k]
```

Comment:
Oops -- but now I'm confused. I thought they are evaluated like Python expressions (https://conda.io/docs/building/meta-yaml.html?highlight=prefix#preprocessing-selectors) so if osx=False and py3k=True then (OS X and py3k) should be False? What expression am I looking for?